git-imerge

```diff
@@ -3717,6 +3749,18 @@ def main(args):
             help='the tip of the branch to be merged into HEAD',
         )
 
+    def add_commit_to_reparent_argument(subparser):
```

Comment:
There's no need for this to be in a function, since it is only called once. (`add_*_option()` functions are used when the same option applies to multiple commands.)